contrib/ovirt/lib/ovirtlago/repoverify.py

```diff
@@ -107,7 +107,13 @@ def fetch_xml(url):
     Returns:
         lxml.etree._Element: Root of the xml tree for the retrieved resource
     """
-    content = urllib2.urlopen(url).read()
+    try:
+        content = urllib2.urlopen(url).read()
+    except urllib2.URLError, e:
```

Comment:
you should use the newer recommended form:

    except urllib2.URLError as error